pkg/test/component_kinesis.go

```diff
@@ -65,9 +66,16 @@ func doRunKinesis(name string, configMap configInput) {
 		},
 		HealthCheck: func() error {
 			kinesisClient := ProvideKinesisClient(name)
-			_, err := kinesisClient.ListStreams(&kinesis.ListStreamsInput{})
 
-			return err
+			_, err = kinesisClient.ListStreams(&kinesis.ListStreamsInput{})
+
+			if err != nil {
+				return err
+			}
+
+			time.Sleep(1 * time.Second)
```

Comment:
😱

Reply:
if we really need to add this sleep here maybe we should add a small comment explaining why it was done.